Detections/ASimAuthentication/imAuthBruteForce.yaml

```diff
@@ -0,0 +1,40 @@
+id: a6c435a2-b1a0-466d-b730-9f8af69262e8
+name: Brute force attack against user credentials (Uses Authentication Normalization)
```

Comment:
As suggested in the other normalization PR.  Should we go with a different start on the name to make the source of this more clear. Something like - "[Normalized Data Result]: Brute force attack against user credentials"

Reply:
Thanks